libraries/SITL/SIM_JSON.cpp

```diff
@@ -323,14 +323,15 @@ void JSON::recv_fdm(const struct sitl_input &input)
 
         airspeed_pitot = state.airspeed;
     } else {
-        // velocity relative to airmass in body frame
-        velocity_air_bf = dcm.transposed() * velocity_ef;
+        
+        // wind is not supported yet for JSON sim, assume zero for now        
+        wind_ef.zero(); 
 
-        // airspeed
-        airspeed = velocity_air_bf.length();
```

Comment:
I see other backends still updating these; why do we no longer need to do that here?

Reply:
The calculation is already done inside update_eas_airspeed(). If left as it is, the variable airspeed will be 
 rewritten anyway. I figure it's better to remove the line to avoid confusion.